src/index.html

```diff
@@ -14,37 +17,83 @@ <h2 class="gallery-title">Our Meetups Gallery</h2>
+            <p class="card-text card-secondary-text text-supporting">
+              <img class="map-icon" src="images/map-pin.svg" /> Bo's Coffee,
+              Ayala
+            </p>
           </div>
         </div>
 
         <div class="card gallery-grid-rightCol">
-          <img class="card-photo" src="images/meetup-july.jpg" alt="meetup-july">
+          <img
+            class="card-photo"
+            src="images/meetup-july.jpg"
+            alt="meetup-july"
+          />
           <div class="card-text-container">
             <p class="card-title">July RoundTable Meetup</p>
-            <p class="card-text card-secondary-text text-supporting"><img class="map-icon" src="images/map-pin.svg"> Coffee Project, Mandaluyong</p>
+            <p class="card-text card-secondary-text text-supporting">
+              <img class="map-icon" src="images/map-pin.svg" /> Coffee Project,
+              Mandaluyong
+            </p>
           </div>
         </div>
-        
+
         <div class="card gallery-grid-leftCol">
-          <img class="card-photo" src="images/meetup-june.jpg" alt="meetup-june">
+          <img
+            class="card-photo"
+            src="images/meetup-june.jpg"
+            alt="meetup-june"
+          />
           <div class="card-text-container">
             <p class="card-title">June RoundTable Meetup</p>
-            <p class="card-text card-secondary-text text-supporting"><img class="map-icon" src="images/map-pin.svg"> Coffee Project, Mandaluyong</p>
+            <p class="card-text card-secondary-text text-supporting">
+              <img class="map-icon" src="images/map-pin.svg" /> Coffee Project,
+              Mandaluyong
+            </p>
           </div>
         </div>
 
         <div class="card gallery-grid-rightCol">
-          <img class="card-photo" src="images/meetup-february.jpg" alt="meetup-february">
+          <img
+            class="card-photo"
+            src="images/meetup-february.jpg"
+            alt="meetup-february"
+          />
           <div class="card-text-container">
             <p class="card-title">February Coffee + Code Meetup</p>
-            <p class="card-text card-secondary-text text-supporting"><img class="map-icon" src="images/map-pin.svg"> Bo's Coffee Primo, Mandaluyong</p>
+            <p class="card-text card-secondary-text text-supporting">
+              <img class="map-icon" src="images/map-pin.svg" /> Bo's Coffee
+              Primo, Mandaluyong
+            </p>
           </div>
         </div>
       </div>
     </section>
+
+    <section class="involve">
+      <h2 class="involve-title">
+        Get Involved
+      </h2>
+      <p class="message">
+        Want to volunteer? Help us to grow or be a speaker and share your
+        knowledge or your learning journey with the community?
+      </p>
+
+      <a
+        class="button-primary"
+        href="https://m.me/freeCodeCamp.Manila"
+        target="_blank"
```

Comment:
Please add `rel="noopener noreferrer"` [here's why](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/).

Reply:
noted on this already resolve this. thanks